app/Http/Controllers/AdministrationController.php

```diff
@@ -204,6 +217,41 @@ private static function getNominationsData()
         return $data;
     }
 
+    private static function getResultsData()
+    {
+        $nominees = Nominee::all();
+        $faculties = Faculty::all();
+        $votes = Vote::all();
+        $data = [];
+
+        foreach ($nominees as $nominee) {
+            $nominee['faculty'] = $faculties->find($nominee['faculty_id']);
+            $nominee['votes'] = $votes->where('nominee_id', $nominee['id'])->where('isFake', 0)->count();
+            array_push($data, $nominee);
+        }
+
+        /*usort(
+        $data,
+        function ($a, $b) {
+        return $a['votes'] - $b['votes'];
+        }
+        );*/
+
+        arsort($data);
+        return $data;
+    }
+
+    private static function getVotersData()
+    {
+        $votes = Vote::all()->whereNotIn('isFake', [1]);
+        $data = [];
+
+        foreach ($votes as $vote) {
+            array_push($data, $vote['voter_email']);
```

Comment:
You will get whole collection from query, why iterate it into new array once again?

Reply:
I want just email

Reply:
Then you could map it similarly as with Select in c#, this is much more inefficient